Resources/Audio/Voice/Reptilian/attritbutions.yml

```diff
@@ -2,3 +2,8 @@
   copyright: '"scream_lizard.ogg" by Skyrat-SS13'
   license:
   source: https://github.com/Skyrat-SS13/Skyrat-tg/pull/892
+
+- files: [reptilian_tailthump.ogg]
+  copyright: "Taken from https://github.com/shiptest-ss13/Shiptest/pull/2159/commits/034844b6946d15a680bf8f9edab2ded7051b305c#diff-5c6dee940bb4ac50e1bb77b17f6a84a083f901a4289823c40852520e8d016edc"
+  license: "CC-BY-SA-3.0"
```

Comment:
This should link to the original file on freesound and have the correct licence
https://freesound.org/people/TylerAM/sounds/389665/